doc/source/schemas/rnaquantification.rst

```diff
@@ -0,0 +1,550 @@
+RnaQuantifications
```

Comment:
I'm not sure if it's more or less helpful to provide the avro generated docs alongside the proto, especially since the fields are renamed in some places.